source/common/router/config_impl.cc

```diff
@@ -560,6 +564,15 @@ RouteEntryImplBase::WeightedClusterEntry::perFilterConfig(const std::string& nam
   return DynamicRouteEntry::perFilterConfig(name);
 }
 
+const PerRouteConfigObject*
+RouteEntryImplBase::WeightedClusterEntry::perFilterConfigObject(const std::string& name) const {
+  const PerRouteConfigObject* cfg = per_filter_configs_.get_object(name);
+  if (cfg != nullptr) {
```

Comment:
nit: Use ternary operator here. Can you fix other cases of this? I think @rodaine might have another one that I didn't comment on.